packages/archway-proto/.npmrc

```diff
@@ -0,0 +1 @@
+scripts-prepend-node-path=true
```

Comment:
Please, remove npm-specific files, as this repo should be built using Yarn 2 instead.